### ISSUE TITLE
Multi-camera draw the sky multiple times

### ISSUE THREAD

**Issue by zhuxudong** [oasis-engine/engine#490]
**Describe the bug**
Multi-camera draw the sky multiple times

**To Reproduce**
```
// Create camera
const cameraEntity = rootEntity.createChild("camera_entity");
cameraEntity.transform.setPosition(0, 0, 50);
cameraEntity.addComponent(OrbitControl);
const camera = cameraEntity.addComponent(Camera);
camera.cullingMask = ~Layer.Layer1;

// GUI camera
const guiCameraEntity = rootEntity.createChild();
const orthoCamera = guiCameraEntity.addComponent(Camera);

guiCameraEntity.transform.setPosition(0, 0, 1);
orthoCamera.isOrthographic = true;
orthoCamera.cullingMask = Layer.Layer1;
orthoCamera.orthographicSize = engine.canvas.height / 2;
orthoCamera.clearFlags = CameraClearFlags.Depth;

```

![image](https://user-images.githubusercontent.com/17639043/132633052-81908b1e-eb1e-4095-aaaa-10cef84e02c8.png)

**Expected behavior**

**Comment by gz65555** [comment]
Should `scene.background` add a `layer` property to enable culling?

**Comment by zhuxudong** [comment]
have fixed in #675
